keras_nlp/applications/bert.py

```diff
@@ -66,118 +67,118 @@ class BertModel(keras.Model):
     def __init__(
         self,
         vocab_size,
-        num_layers=12,
-        hidden_size=768,
-        dropout=0.1,
-        num_attention_heads=12,
-        inner_size=3072,
+        num_layers,
+        hidden_size,
+        num_attention_heads,
+        inner_size,
         inner_activation="gelu",
         initializer_range=0.02,
+        dropout=0.1,
         max_sequence_length=512,
         type_vocab_size=2,
```

Comment:
we are calling the input `segments` so we should probably name this to agree. `num_segments` maybe?